source/common/http/conn_manager_impl.h

```diff
@@ -290,16 +327,32 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     ActiveStream(ConnectionManagerImpl& connection_manager);
     ~ActiveStream();
 
+    // Indicates which filter to start the iteration with.
+    enum class FilterIterationStartState { Always_start_from_next, Can_start_from_current };
+
     void addStreamDecoderFilterWorker(StreamDecoderFilterSharedPtr filter, bool dual_filter);
     void addStreamEncoderFilterWorker(StreamEncoderFilterSharedPtr filter, bool dual_filter);
     void chargeStats(const HeaderMap& headers);
+    // Returns the encoder filter to start iteration with. If the function is called from a filter
```

Comment:
nit: now with the enum I think the 2nd part of this comment is obvious and can be removed, same below.

Reply:
Fixed. Thanks!